### PR TITLE
fix #240

### LINKED ISSUE #240: support omit_graph flag
The [omit_graph flag](https://json-ld.org/spec/latest/json-ld-framing/#dfn-omit-graph-flag) is quite important for clients who consume jsonld as json (like, I suppose, most web annotation clients). I'll open an PR for that

### PR DESCRIPTION
**Pull request by eroux**
there is a pending question for the tests, see https://github.com/json-ld/json-ld.org/issues/676

### CODE REVIEW

**Comment by eroux** [comment]
I think this is ready to be merged, I've updated the test expectations with https://github.com/w3c/json-ld-framing/pull/11

File: core/src/test/resources/json-ld.org/frame-manifest.jsonld [hunk 1]
```diff
@@ -167,10 +167,20 @@
       "input": "frame-0030-in.jsonld",
       "frame": "frame-0030-frame.jsonld",
       "expect": "frame-0030-out.jsonld"
+    }, {
+      "@id": "#tg001",
+      "@type": ["jld:PositiveEvaluationTest", "jld:FrameTest"],
+      "name": "Library framing example with @graph and omitGraph is true.",
+      "purpose": "Basic example used in playground and spec examples.",
+      "input": "frame-g001-in.jsonld",
```
Comment (ansell) [inline core/src/test/resources/json-ld.org/frame-manifest.jsonld]: These test files need to be checked into git

**Comment by eroux** [comment]
done, sorry for that!

**Comment by ansell** [comment]
np, merged it now

**Pull request merged**